app/src/main/res/layout/article_list_item_view.xml

```diff
@@ -3,25 +3,49 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:orientation="horizontal"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="100dp"
     android:paddingStart="10dp"
+    android:paddingEnd="10dp"
     android:paddingTop="10dp"
-    android:paddingBottom="10dp">
+    android:paddingBottom="10dp"
+    android:weightSum="1">
 
-    <TextView
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_weight="1"
-        android:textAppearance="?android:attr/textAppearanceMedium"
-        android:text="headline"
-        android:id="@+id/headline" />
 
-    <TextView
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:textAppearance="?android:attr/textAppearanceSmall"
-        android:text="byline"
-        android:id="@+id/byline"
-        android:layout_weight="1" />
+    <ImageView
+        android:layout_width="100dp"
+        android:layout_height="50dp"
+        android:id="@+id/lead_art"
+        android:layout_gravity="start|center_vertical"
+        android:src="#0c14ff" />
+
+    <LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingStart="10dp"
+        android:paddingEnd="10dp"
+        android:paddingTop="5dp"
+        android:paddingBottom="5dp"
+        android:layout_weight="0.09">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:textAppearance="?android:attr/textAppearanceMedium"
```

Comment:
do you understand what's happening here?  also, if you have time, check out "styles"...  i'll give you a quick penny tour...

in `styles.xml` (google where to save it), you can define sets of XML attributes, and inherit from other styles, like so

```
<style name="MyAppTextStyle">
  <item name="android:layout_width">match_parent</style>
  <item name="android:layout_height">wrap_content</style>
</style>
<style name="MyAppBigTextStyle" parent="MyAppTextStyle"><!-- note the "parent", will get all attributes in the parent -->
  <item name="android:textSize">14sp</item>
</style>
```

then you use it for the global `style` attribute, reference it like any other value `@style/NameOfYourStyle`

```
<TextView
  android:id="@+id/bob"
  style="@style/MyAppBigTextStyle" />
```